website/docusaurus.config.js

```diff
@@ -1,199 +1,39 @@
-          },
-          {
-            href: 'https://github.com/visgl/react-map-gl',
-            label: 'GitHub',
-            position: 'right'
-          }
-        ]
-      },
-      footer: {
-        style: 'dark',
-        links: [
-          {
-            title: 'Resources',
-            items: [
-              {
-                label: 'API Reference',
-                to: '/docs/api-reference/map'
-              },
-              {
-                label: 'Starter templates',
-                href: 'https://github.com/visgl/react-map-gl/tree/master/examples/get-started'
-              }
-            ]
-          },
-          {
-            title: 'Other vis.gl Libraries',
-            items: [
-              {
-                label: 'deck.gl',
-                href: 'https://deck.gl'
-              },
-              {
-                label: 'luma.gl',
-                href: 'https://luma.gl'
-              },
-              {
-                label: 'loaders.gl',
-                href: 'https://loaders.gl'
-              },
-              {
-                label: 'kepler.gl',
-                href: 'https://kepler.gl'
-              }
-            ]
-          },
-          {
-            title: 'More',
-            items: [
-              {
-                label: 'Open Visualization',
-                href: 'https://www.openvisualization.org/#'
-              },
-              {
-                label: 'vis.gl blog on Medium',
-                href: 'https://medium.com/vis-gl'
-              },
-              {
-                label: 'GitHub',
-                href: 'https://github.com/visgl/react-map-gl'
-              }
-            ]
-          }
-        ],
-        copyright: '<p>Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and vis.gl contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS Foundation</a> has registered trademarks and uses trademarks.  For a list of trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  Trademarks and logos not indicated on the <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation trademarks</a> are trademarks&trade; or registered&reg; trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.</p><p><a href="https://openjsf.org">The OpenJS Foundation</a> | <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | <a href="https://bylaws.openjsf.org">Bylaws</a> | <a href="https://code-of-conduct.openjsf.org">Code of Conduct</a> | <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | <a href="https://trademark-list.openjsf.org">Trademark List</a> | <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a></p>'
```

Comment:
Is this included by default now?

Reply:
Good catch, no. I will update ocular to allow customization.